stepman/paths_test.go

```diff
@@ -0,0 +1,89 @@
+package stepman
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path"
+	"testing"
+)
+
+const (
+	GivenSteplibURI  = "https://github.com/bitrise-io/steplib"
+	GivenFolderAlias = "12334556"
+	GivenHomePath    = "/usr/testeruser/"
+	GivenStepID      = "test-custom-step"
+	GivenStepVersion = "0.5.6"
+)
+
+func GivenRoute() SteplibRoute {
+	return SteplibRoute{
+		SteplibURI:  GivenSteplibURI,
+		FolderAlias: GivenFolderAlias,
+	}
+}
+
+func Test_GivenHomeDir_WhenGetStepmanDirPathCalled_ThenGoodPathReturned(t *testing.T) {
+	// Given
+	err := os.Setenv("HOME", GivenHomePath)
```

Comment:
Just thinking, wouldn't it mess up the `HOME` env running this test?

Reply:
@trapacska wdyt? Not sure how we could test this

Reply:
Ok I added logic to reset the HOME variable after the test

Reply:
Btw, we've looked into this and it will not overwrite the variable as this change is visible only in this process. But, nevertheless, it's a good pattern to clean up after each test, so I'll leave it.